pkgs/development/python-modules/staticjinja/default.nix

```diff
@@ -8,17 +8,18 @@
 
 buildPythonPackage rec {
   pname = "staticjinja";
-  version = "0.3.5";
+  version = "0.4.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "fbd61cca1dad44b6891d1a1d72b11ae100e21b3909802e3ff1861ab55bf16603";
+    sha256 = "0v535dq6mli5a1abd01kxg0bjfcnfw2cvs8fd1cr6h08j24kfy2r";
   };
 
-  propagatedBuildInputs = [ jinja2 docopt easywatch ];
-
-  # There are no tests on pypi
-  doCheck = false;
+  propagatedBuildInputs = [
+    jinja2
+    docopt
+    easywatch
+  ];
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
I managed to fix the unit tests! The trick was using the source from github _and_ ignoring path mismatches